### PR TITLE
001 view job roles

### PR DESCRIPTION
**Pull request by Ryan-Jones1999**

### CODE REVIEW

File: tests/Unit/Service/employeeservicetest.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    
+            var results = await EmployeeService.viewjobroles();
+            
+            expect(results).to.deep.equal(jobroles)
+          })
+
+          it('should throw exception when 500 error returned from axios when calling viewJobRoles', async () => {
+            var mock = new MockAdapter(axios);
+    
+            const data = jobroles;
```
Comment (hellofadude) [inline tests/Unit/Service/employeeservicetest.js:31]: Remove unused variable

File: tests/Unit/Service/employeeservicetest.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+var axios = require('axios');
+var MockAdapter = require('axios-mock-adapter');
+
+var chai = require("chai");
```
Comment (hellofadude) [inline tests/Unit/Service/employeeservicetest.js:4]: Use `const` or `let` in line with ECMAScript 6

**Review by hellofadude — APPROVED** [review]
Approved!

**Review by asadzafark — APPROVED** [review]
nice

**Pull request merged**